src/IndexManager.php

```diff
@@ -151,7 +158,7 @@ public function rawSearch($query, $className, $page = 1, $nbResults = null, arra
             $nbResults = $this->configuration['nbResults'];
         }
 
-        return $this->engine->search($query, $this->getFullIndexName($className), $page, $nbResults, $parameters);
+        return $this->engine->search($query, $this->getFullIndexName($className), $page,  $nbResults, $parameters);
```

Comment:
extra space here that you don't need :-)

Reply:
Thanks, fix